code/game/objects/effects/landmarks.dm

```diff
@@ -3,98 +3,199 @@
 	icon = 'icons/mob/screen_gen.dmi'
 	icon_state = "x2"
 	anchored = 1.0
+	layer = 4
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
 
-/obj/effect/landmark/New()
-
-	..()
+/obj/effect/landmark/Initialize(mapload)
```

Comment:
This can break away missions I think. They are loaded when the world is Newed at this moment by `SSmapping`. Which is before `Initialize` is ever fired

Reply:
Do away missions make use of landmarks for their loading? I assumed anything within could be loaded on initialize later.

Reply:
Gateways should be compatible with Initialize, since players cant even interact with it before game start, and SSatoms uses that.


The gateway loads just after the station in SSmapping, and given the station landmarks dont have these issues, the gateway ones wont.

Reply:
I was referring to code in `/datum/controller/subsystem/mapping/proc/load_away_mission()`
![image](https://user-images.githubusercontent.com/15887760/146545742-cafd92e9-63ac-410b-91bd-c1d8c0d5e1af.png)

This will get hit when you load in for example the terror spider gateway. It will then use these landmarks to plop you down at when you jump into the gateway without the other side being active.

I just tested this in your PR and the breakpoint there doesn't get hit since SSatoms inits after SSmapping (which is logical)
Causing this runtime when you bump into the gateway:
![image](https://user-images.githubusercontent.com/15887760/146546997-2fc3676b-0343-4b47-9cae-fc0854332f1b.png)

Reply:
Do the immediate initialize part here